types/simpl-schema/index.d.ts

```diff
@@ -122,6 +122,7 @@ interface SimpleSchemaStatic {
   addValidator(validator: () => boolean): any;
   pick(...fields: string[]): SimpleSchemaStatic;
   omit(...fields: string[]): SimpleSchemaStatic;
+  oneOf(...types: Array<(SchemaDefinition | StringConstructor | StringConstructor | NumberConstructor | DateConstructor | ArrayConstructor)>): SimpleSchemaStatic;
```

Comment:
StringConstructor is listed twice here. Maybe you meant to have BooleanConstructor instead?

Reply:
My bad, I replaced the first one with BooleanConstructor.